tests/tests_auth.py

```diff
@@ -89,7 +96,47 @@ async def endpoint(request: Request) -> Response:
         self.assertEqual('user', user.username, "Wrong username for user")
 
         self.assertEqual(
-            ['authenticated', 'blueshirt'],
+            ['authenticated', BLUESHIRT_SCOPE],
+            scopes,
+            "Wrong scopes for user",
+        )
+
+
+class FileAuthTests(test_utils.AsyncTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.backend = FileBackend(
+            path=Path(__file__).parent / 'fixtures' / 'auth-file.yml'
```

Comment:
I think the linters will want a trailing comma here?

Reply:
https://github.com/PeterJCLaw/code-submitter/pull/15/commits/a6bb0f96418638c0a32631d8e249649c0e4780cb